app/controllers/registrations_controller.rb

```diff
@@ -1,18 +1,26 @@
-class Contractors::RegistrationsController < Devise::RegistrationsController
+class RegistrationsController < Devise::RegistrationsController
   before_action :configure_permitted_parameters, if: :devise_controller?
```

Comment:
I'm not sure if you need this `before_action` 

1. comment out the line
2. run `rake`
3. If the tests ar still green - remove code.